about-static-page

nataliyah123 opened the pull request:
@mattwr18 . please take a look at the ticket no #102 . sponsors and css is missing.what do you suggest we should do

fixes #102

nataliyah123 committed:
about-static-page

mattwr18 commented on src/components/homepage/About.js:
```diff
@@ -0,0 +1,46 @@
+	constructor(props){
+		super(props);
+		this.state={about: null}
+	}
```
this can be refactored to `state = { about: null }`

mattwr18 commented on src/components/homepage/About.js:
```diff
@@ -0,0 +1,46 @@
+				   return( <RingLoader sizeUnit={'px'} size={200} color={'#34495E'} />)
+				
+			     }	
+				
```
maybe we could add `Container` around the `div` and `RingLoader` so that we have some padding and for consistency?

mattwr18 commented:
hey @nataliyah123 great job getting this started! 
I have a few thoughts... I think we don't want to have the `About` component in `/components/homepage`
It is a stateful component, which we have been trying to put in the `containers` directory

Also, this is failing the build with this message:
```
yarn test
yarn run v1.3.2
$ export NODE_ENV=testing && standard && jest
standard: Use JavaScript Standard Style (https://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
```
we have a `npm script` called `fix`, so you could try running `yarn fix`
if you try running the tests locally with `yarn test`, it will tell you if you have linting issues that will cause the Travis CI build to fail

nataliyah123 committed:
static about page r2

nataliyah123 commented:
@mattwr18  made the changes

nataliyah123 committed:
About static page r3

nataliyah123 committed:
Merge branch 'develop' of https://github.com/AgileVentures/WebsiteOne-FE into 102-About-page

nataliyah123 commented:
@mattwr18 . now it is failing the ci build because the version of navbar i have, does not have the getting started option . locally it is clearing all test because i have added one more menu item and changed the navbar test a little bit .

joaopapereira commented on src/tests/components/Navbar.test.js:
```diff
@@ -13,14 +13,14 @@ describe('Navbar', () => {
     })
 
-    it('renders 6 MenuItem elements', () => {
+    it('renders 7 MenuItem elements', () => {
```
I am not 100% sure that this test make sense anymore. My vote is to remove the this test and the one bellow that check the links rendered

mattwr18 replied:
Sorry, I didn't mean to mark this as resolved... I wanted to make a comment... I agree with you @joaopapereira... it's annoying to need to update this every time we add another link. 
Also, we will be testing that the link exists by clicking on it in cypress tests

joaopapereira commented on src/containers/About.js:
```diff
@@ -0,0 +1,28 @@
+    super(props)
+    this.state = { about: null }
+  }
+  componentDidMount () {
```
```suggestion

  componentDidMount () {
```

joaopapereira reviewed (changes requested):
@nataliyah123 great to see you contributing to this project 😄 
I addded some comments on the PR when you can please review them

nataliyah123 committed:
Update About.js

added a line before componentdidmount

joaopapereira committed:
Update src/containers/About.js

Co-Authored-By: nataliyah123 <nataliyah123@users.noreply.github.com>

nataliyah123 committed:
Update About.js

mattwr18 commented on src/containers/About.js:
```diff
@@ -0,0 +1,29 @@
+import '../components/homepage/Homepage.css'
```
is there a reason we are importing this css here @nataliyah123?
I don't think we need this, but you have more context than I do so if there is one, could you maybe explain it?

nataliyah123 committed:
static About page r4

nataliyah123 committed:
static about r4

nataliyah123 committed:
static about r4

nataliyah123 committed:
static about page r5

nataliyah123 committed:
About page r5
